### ISSUE TITLE
Enable configuration of barcode & ASN parsing

### ISSUE THREAD

**Issue by sahnetorti** [BenoitAnastay/paperless-home-assistant-addon#35]
I have started using paperless via this add-on and like both, paperless and the add-on. Thanks for all the work!

I am a bit of a loss to where to ask the proper question, if this is not the right place maybe you can point me towards it :)
I would like to use the automatic ASN barcode/QR code scanning feature of paperless. In the original paperless documentation it is mentioned that the options PAPERLESS_CONSUMER_ENABLE_BARCODES and PAPERLESS_CONSUMER_ENABLE_ASN_BARCODE must be set to "true" and default to "false".
I am at a loss how to set these options in the context of the add-on.
Any help appreciated.

**Comment by BenoitAnastay** [comment]
I'll add those options in the next release tomorrow

**Comment by sahnetorti** [comment]
That would be great, thanks :)

**Comment by BenoitAnastay** [comment]
@sahnetorti 
I'm refractoring the Add-on, you can check  on the edge repository if it's working https://github.com/BenoitAnastay/home-assistant-addons-repository-edge

**Comment by sahnetorti** [comment]
@BenoitAnastay I did a quick test this morning and can confirm that the barcode scanning works.
I was able to toggle the two switches in the configuration UI and afterwards ASN barcodes were recognised by paperless.

They needed to be big enough it seems, as these [QR codes](https://margau.net/posts/2023-04-16-paperless-ngx-asn/pdf/labels4731_001.pdf) did not work (via reference from [margau.net](https://margau.net/posts/2023-04-16-paperless-ngx-asn/). When using bigger 1D barcodes things were working. I guess this has to do with paperless itself and not with your add-on.

I did not test the new version beyond the barcode scanning, althoug paperless seems to work fine (simple navigation, document editing, saved searches).
 
Many thanks for your dedication!
